packages/terser/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.base.json",
   "include": ["src/**/*", "types/**/*"],
   "compilerOptions": {
+    "module": "ES2020",
```

Comment:
Had to set this flag, otherwise the TypeScript compiler will start complaining about using `import.meta.url` in something that compiles to a CommonJS target. Which of course is a non-issue as Rollup is used to compile the code, not the TypeScript compiler.

That said, should this not be the default? Then opting back into CommonJS where necessary (not even sure if that is needed).